tests/ludwig/marshmallow/test_fields_preprocessing.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+
+
+import ludwig.schema.preprocessing as lsp
+
+
+def get_marshmallow_from_dataclass_field(dfield):
+    """Helper method for checking marshmallow metadata succinctly."""
+    return dfield.metadata["marshmallow_field"]
+
+
+def test_PreprocessingDataclassField():
```

Comment:
Let's stick with snake case that we've been using for all of tests so far.

Reply:
Good catch, will make the change!

Reply:
Done!